pytest_splunk_addon/fields_tests/test_templates.py

```diff
@@ -23,6 +23,7 @@
 from ..addon_parser import Field
```

Comment:
Maybe we could keep the proper order of imports? (move `from ..addon_parser import Field` to line 27, and remove blank line 29)?

Reply:
okay, done